### ISSUE TITLE
Tech debt: rename rollapp.LastStateUpdateHeight to something more suitable

### ISSUE THREAD

**Issue by danwt** [dymensionxyz/dymension#1356]
After #1343 it will not always correspond to last state update height
I avoided changing it for now to avoid proto conflicts

**Comment by danwt** [comment]
superseded by https://github.com/dymensionxyz/dymension/issues/1357
